examples/tutorials/DeepChemXAlphafold.ipynb

```diff
@@ -0,0 +1,1767 @@
+{
```

Comment:
<p>Make this a "Save to Google Drive" section</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/3574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
done